Sources/AppcuesKit/Data/Analytics/AutoPropertyDecorator.swift

```diff
@@ -55,38 +68,28 @@ internal class AutoPropertyDecorator: AnalyticsDecorating {
             sessionLatestUserProperties = [:]
             currentScreen = nil
             previousScreen = nil
-        case .group(nil):
-            // removing from a group should not have any auto props
-            return tracking
-        case .group:
-            // group updates only have this single auto prop, so add that and return early
-            decorated.properties = (decorated.properties ?? [:]).merging(["_lastSeenAt": Date()])
-            return decorated
         default:
             break
         }
 
         let now = Date()
 
-        var sessionProperties: [String: Any?] = [
+        let sessionProperties: [String: Any?] = [
             "userId": storage.userID,
             "_isAnonymous": storage.isAnonymous,
             "_localId": storage.deviceID,
             "_sessionPageviews": sessionPageviews,
             "_sessionRandomizer": sessionRandomizer,
             "_currentScreenTitle": currentScreen,
             "_lastScreenTitle": previousScreen,
+            "_lastBrowserLanguage": Bundle.main.preferredLocalizations.first,
```

Comment:
If this value is `nil` it gets filtered out by `compactMapValues` below, so no change in behaviour, and now `sessionProperties` can be `let`.